src/main/java/frc/robot/subsystems/shooter/commands/bits/TestShooterVelocity.java

```diff
@@ -0,0 +1,33 @@
+package frc.robot.subsystems.shooter.commands.bits;
+
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.ParallelRaceGroup;
+import edu.wpi.first.wpilibj2.command.RunCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.subsystems.shooter.Shooter;
+
+public class TestShooterVelocity extends SequentialCommandGroup {
+
+    public TestShooterVelocity(Shooter shooter) {
+
+        addCommands(
+                        new ParallelRaceGroup(
+                                new InstantCommand(
```

Comment:
```suggestion
                                new RunCommand(
```
Also for the other parallel commands